specification/batch/resource-manager/Microsoft.Batch/2017-05-01/BatchManagement.json

```diff
@@ -1496,6 +1544,44 @@
           "title": "The URL to get the next set of operation list results if there are any."
         }
       }
+    },
+    "CheckNameAvailabilityParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name to check for availability"
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource type. Must be set to Microsoft.Batch/batchAccounts"
```

Comment:
Should this be modeled as enum with one default value? like: https://github.com/Azure/azure-rest-api-specs/blob/current/specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json#L689 ?